### ISSUE TITLE
Connected car details

### ISSUE THREAD

**Issue by chemicalandy** [dan-r/HomeAssistant-Ohme#64]
This integration is great and it's made it easy for me to stop by house battery from discharging when the car is charging. (I'm also using your Nissan Leaf integration which is equally good).

I have 2 different EVs and need to charge one of them most days. This means that I frequently need to change the car details and charge schedule in the Ohme app before plugging in. It would be helpful to me if it was possible to do this via the integration as I could then save the details of each car and a default schedule for each in Home Assistant and then automate changing the car and schedule.

I'm not sure if this is possible or of interest to enough people, but I thought it was worth asking.

Andy

**Comment by dan-r** [comment]
Thanks for your comment! Apologies for not getting round to replying sooner.

I've deliberately not done this yet, but its on the roadmap.

Supposedly there is a new version of the Ohme app coming in the next couple of months which natively supports multiple vehicles. I'm hoping by waiting for this and then integrating with it, adding the feature will be more 'native' and lower effort in terms of development and maintenance. 

I'll leave this open to track it, and will try to get started as soon as I get my hands on the new app!

**Comment by hogsterbogster** [comment]
This would be a fantastic addition to an already excellent integration!

When I’m not fast charging my two EVs overnight at their full charge rate, I use the vehicle sub-category ‘Solar Cars’ and pick the relevant ‘car’ model I want to use based on its charge rate in kW depending on how quick I want to charge the car (depending on how much our solar panels are generating).

If the car make / model could be accessed from HA I could then link the car model to a charge power slider that would save me faffing about going into the Ohme app and selecting a different ‘car’ each time I wanted to change the charge current.

Thanks for all your efforts on this! 😎👍🏻🙏

David

**Comment by dan-r** [comment]
Charge power limiting is something I'd also like to add, but again waiting on the new app before reinventing the wheel :)

Part of my hesitation with adding it to the integration before the app supports it is the user experience. It would be much neater to setup all your cars in the app then be able to switch from Home Assistant rather than having to setup cars with configuration files etc

**Comment by hogsterbogster** [comment]
Agreed 😀 Hurry up Ohme! 

Thanks again,

David

**Comment by chemicalandy** [comment]
> Thanks for your comment! Apologies for not getting round to replying sooner.
> 
> I've deliberately not done this yet, but its on the roadmap.
> 
> Supposedly there is a new version of the Ohme app coming in the next couple of months which natively supports multiple vehicles. I'm hoping by waiting for this and then integrating with it, adding the feature will be more 'native' and lower effort in terms of development and maintenance.
> 
> I'll leave this open to track it, and will try to get started as soon as I get my hands on the new app!

Thanks for the reply. I hadn't realised that Ohme were planning to add this to their new app, and it it's clearly best to wait for them to do this.

**Comment by gee-jay-bee** [comment]
Agreed this would be a great addition to an already great integration!

I also have two cars but I'm typically having a PHEV connected most nights and occasionally switching to the other. My hope for this integration when I came to raise this enhancement request was more modest ... that the details of the connected car could be surfaced via a sensor so I could run an automation upon detecting that the car is connected that would send me a notification if the connected car (in Ohme) disagrees with the status of the car itself (in the individual integrations for the cars). That would remind me to change the connected car in the Ohme app. Now, if this integration also offered the ability to set the car then I could take me out of the loop which would be even better!

I note that the car support in the new version of the Ohme app still requires us to search for brand then model. I find it annoying, it would have been nice to add my two cars and simply switch between them!

**Comment by dan-r** [comment]
Closing this issue due to discontinuing this integration in favour of the Home Assistant Core version.

Good news though... Vehicle switching has just become available in the Ohme app so I'm working on adding this to the Core version of the integration in the next week or so.
